Gems/Atom/Utils/Code/Include/Atom/Utils/ImGuiCpuProfiler.h

```diff
@@ -213,6 +213,11 @@ namespace AZ
 
             // Index into the file picker, used to determine which file to load when "Load File" is pressed.
             int m_currentFileIndex = 0;
+
+
+            // --- Loading capture state ---
+            AZStd::unordered_set<AZStd::string> m_deserializedStringPool;
+            AZStd::unordered_set<RHI::CachedTimeRegion::GroupRegionName, RHI::CachedTimeRegion::GroupRegionName::Hash> m_deserializedGroupRegionNamePool;
```

Comment:
We maintain our own string pools and group region name pools here since the `CpuProfilerImpl::InsertDynamicName` only accepts dynamic region names, if we accepted dynamic group + region names then that could be used instead. a